src/__tests__/Search.test.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import {
+  render, screen, fireEvent, waitFor,
+} from '@testing-library/react';
+import { MemoryRouter } from 'react-router-dom';
+import AnimeSearchPage from '../components/SearchPage';
+
+describe('AnimeSearchPage component', () => {
+  it('should show search results when the search button is clicked', async () => {
+    render(
+      <MemoryRouter>
+        <AnimeSearchPage />
+      </MemoryRouter>,
+    );
+
+    const searchInput = screen.getByPlaceholderText(/search for anime/i);
+    const searchButton = screen.getByRole('button', { name: /search/i });
+
+    fireEvent.change(searchInput, { target: { value: 'noblesse' } });
+    fireEvent.click(searchButton);
+
+    await waitFor(() => {
+      const searchResults = screen.getByRole('list');
+      expect(searchResults).toBeInTheDocument();
+    });
+  });
+
```

Comment:
- [ ] I'm sorry to break it but it still seems like this test is failing :pleading_face:  :point_down: 
![Screenshot from 2023-07-08 20-06-05](https://github.com/MoyasiGinko/React-Capstone/assets/99715640/3a285ebe-cc19-4ffb-ba01-12288fc0049c)